ohif-configurable/hp-extension/src/hp/hpMN.ts

```diff
@@ -37,19 +37,6 @@ export default [
             columns: 2,
           },
         },
-        displaySets: [
```

Comment:
This is just cleanup to agree with updated hanging protocol definition, does not affect the HP